recipes-core/images/mistysom-image.bbappend

```diff
@@ -73,3 +73,9 @@ IMAGE_INSTALL_append = " docker-ce"
 # For NodeJS and NPM
 IMAGE_INSTALL_append = " nodejs"
 IMAGE_INSTALL_append = " nodejs-npm"
+
+# Debug
+IMAGE_INSTALL_append = " gdb gdbserver"
+IMAGE_INSTALL_append = " rsync openssh-sftp-server"
+IMAGE_INSTALL_append = " gcc"
+IMAGE_INSTALL_append = " meson ninja cmake make"
```

Comment:
mhm, would it make sense to have two images:
1. mistysom-imagedbg 
2. mistysom-imageprod
or similar? They'd be the same excepr for the dbg image a section might get appended to the bbappend file with all the debugging stuff (I'm also thinking about the PHY bring up tools)

Reply:
Yes, I like it! Easy enough to implement! If we have:
```
IMAGE_INSTALL_append = " gcc"
IMAGE_INSTALL_append = " meson ninja cmake make"
```
do you think it would make sense to add `git` to it as well, maybe?

Reply:
Sure